python/pyspark/errors/__init__.py

```diff
@@ -0,0 +1,33 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from pyspark.errors.errors import *  # noqa: F403
+from pyspark.errors.exceptions import PySparkException
+
+__all__ = [  # noqa: F405
+    "PySparkException",
+    "columnInListError",
+    "higherOrderFunctionShouldReturnColumnError",
+    "notColumnError",
```

Comment:
Exporting a function that returns an instance of an error seems weird and indicates that the constructor is not well designed.